### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Select a Wizard_Screen Reader: Narrator and NVDA are not reading "Wizards" label

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4283]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS1.3.1; #ErrorSuggestion; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2 

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211 

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator , NVDA

**MAS Violated:** MAS1.3.1

**NOTE:** Same issue is observed in the below screen as well 
File > New > Project > Select a Wizard

**Repro Steps:**
1.Start Screen Reader. Open Java EE - Eclipse application
2.Install "Azure Toolkit for Eclipse" and "Scala Plug-ins" from Eclipse Marketplace
3.Tab till "File > New > Project" in the navigation bar and press enter
4.Tab till "HDInsight Project > Spark on HDInsight (Scala)" and select it.
5.Tab till "Next" button and press enter 
6.Enter all the required details, tab till "Next" button and press enter
7.Tab till "Module Dependencies" tab and press enter
8.Tab till "Finish" button and press enter
9.In Project Explorer, expand the created Scala project right-click src, New-->Other and press enter
10.Tab till "Wizards" text box and verify if the screen reader is reading the associated label

**Actual Result:**
Narrator and NVDA are not reading "Wizards" label

**Expected Result:**
Narrator and NVDA should read "Wizards" label

**User impact:**
Screen Reader users will not understand the purpose of the text-box if the label is not read

**MAS Reference:**
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&action=default&mobileredirect=true&cid=8b75f338-0916-41c9-9fdf-ddb91b2b7698)

![Bug_4283_Java Tooling_New Project_Select Wizard_Labels_NVDA](https://user-images.githubusercontent.com/41108985/78988441-8ce98080-7b4e-11ea-84d3-3533ad89290a.png)

![Bug_4283_Java Tooling_New _Select Wizard_Labels_NVDA](https://user-images.githubusercontent.com/41108985/78988497-b60a1100-7b4e-11ea-9923-2cc15158a0dd.png)

**Comment by Flanker32** [comment]
@soumyatippapuram The new peoject wizard is controlled by Eclipse, which is out of the scope of our toolkit, here is the sceen shot of Eclipse without out toolkits, which behaves the same

![image](https://user-images.githubusercontent.com/12445236/79306697-474efe00-7f28-11ea-8254-fd4ad44616e5.png)

**Comment by Flanker32** [comment]
* Importance of experience: Low, users may be confused by an empty label, but this didn't break the workflow.
* Quality of experience: Not impacted. Label "Wizards" didn't tell much information about the text field, instead we could see NVDA do speak the hint message "type filter text", this should be enough for users to understand the usage of the textfield.
* Alternate solutions:  This issue will not break users experience, no need for alternate solutions
* Evidence: All evidence is provided in this issue.
* [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@soumyatippapuram  This issue has been linked to AAT, please have a review, thanks!

**Comment by soumyatippapuram** [comment]
@Flanker32 As per the updated [External Bug Process](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx), we need to route this issue to eDAD Team. We would require a sample app to do so. Request you to provide the sample app.

**Comment by Flanker32** [comment]
@soumyatippapuram Thanks for your help and sorry for the late reply. Do you mean a sample Eclipse project to reproduce this issue?  This issue could be found in Eclipse UI without any project, or is there any other information we need provide?

**Comment by Alexandria-R** [comment]
Product team needs to file a bug on eclipse. Once they have done so, this can be closed as non blocking.

**Comment by Flanker32** [comment]
The external bug is tracked here https://bugs.eclipse.org/bugs/show_bug.cgi?id=562547

**Comment by soumyatippapuram** [comment]
As external tracking bug is available, closing this issue

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749654
